tests/backbeat/tests/api/metrics.js

```diff
@@ -1,191 +1,202 @@
+        next => scalityUtils.createVersionedBucket(srcBucket, next),
+        next => scalityUtils.putBucketReplicationMultipleBackend(srcBucket,
+            'placeholder', roleArn, storageClass, next),
     ], done));
 
-    it('should get bucket-level throughput metrics', done => {
-        let versionId;
-        const destinationKey = `${srcBucket}/${key}`;
+    afterEach(done => parallel([
+        next => scalityUtils.deleteVersionedBucket(srcBucket, next),
+        next => awsUtils.deleteAllVersions(awsDestBucket, destKeyPrefix, next),
+        next => scalityUtils.deleteAllBlobs(destContainer, destKeyPrefix, next),
+    ], done));
 
+    it.skip('should report metrics when replication occurs', done => {
+        const prevCompletions = {};
+        const prevThroughput = {};
         series([
-            next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(10000),
-                (err, res) => {
+            next => {
+                // first wait for any previous backlog to complete
+                const expectedBody = {
+                    backlog: {
+                        description: 'Number of incomplete replication ' +
+                        'operations (count) and number of incomplete bytes ' +
+                        'transferred (size)',
+                        results: { count: 0, size: 0 },
+                    },
+                };
+                getAndCheckResponse(`${pathPrefix}/all/backlog`, expectedBody,
+                    next);
+            },
+            next => makeGETRequest(`${pathPrefix}/all`, (err, res) => {
+                assert.ifError(err);
+                getResponseBody(res, (err, body) => {
                     assert.ifError(err);
-                    versionId = res.VersionId;
+                    prevCompletions = body.completions.results;
+                    prevThroughput = body.throughput.results;
                     next();
-                }),
-            next => scalityUtils.compareObjectsAWS(srcBucket, destBucket, key,
-                undefined, next),
-            next => makeDelayedGETRequest('/_/backbeat/api/metrics/crr/' +
-                `${destLocation}/throughput/${destBucket}/${destinationKey}` +
-                `?versionId=${versionId}`,
-                (err, res) => {
-                    assert.ifError(err);
-                    assert.equal(res.statusCode, 200);
-
-                    getResponseBody(res, (err, body) => {
-                        assert.ifError(err);
-                        assert(body.description);
-                        assert(body.throughput);
-
-                        next();
-                    });
-                }),
+                });
+            }),
+            // NOTE: metrics are doubled because 2 destination locations
+            next => scalityUtils.putObject(srcBucket, key, Buffer.alloc(100),
+                next),
+            next => scalityUtils.compareObjectsAWS(srcBucket, awsDestBucket,
```

Comment:
Since this is a one-to-many setup we can just use the util `waitUntilReplicated`—i.e. no need to compare the destination object.